### ISSUE TITLE
Add html , Css or Js Codes(Web Development)

### ISSUE THREAD

**Issue by AbhilashaGupta16** [AbhilashaGupta16/HacktoberFest-#4]

**Comment by nick-bors** [comment]
hey, can you be more specific? you want a website or...

**Comment by ananya0504** [comment]
So recently I designed an introduction page for myself, can I put that here?

**Comment by Ansh04122001** [comment]
can i add to do list code in html css js

**Comment by AbhilashaGupta16** [comment]
> hey, can you be more specific? you want a website or...

**Comment by AbhilashaGupta16** [comment]
> hey, can you be more specific? you want a website or...

**Comment by AbhilashaGupta16** [comment]
yes ananya you can put your code here. It can be anything , simple frontend or a basic design or the whole website

**Comment by ananya0504** [comment]
Can I add my figma links? For UI?

**Comment by hambali999** [comment]
Hey I am interested to help! Are you just looking for a basic website template?

**Comment by hambali999** [comment]
Hey! i have added a simple website for you! you can check it on my pull request :) this is for hacktoberfest2020 thank you!

**Comment by simran1199** [comment]
Hi!! can I add my simple score keeper code for playing games (using HTML, CSS, Javascript)?

**Comment by yashrsb** [comment]
Hey, I would like to add a simple website created in html & css. @AbhilashaGupta16 Please assign it to me.

**Comment by AbhilashaGupta16** [comment]
I am reviewing all your codes, and its open to all anyone can contribute here.

**Comment by Ansh04122001** [comment]
can i add django project file in python

**Comment by yashrsb** [comment]
@AbhilashaGupta16 Please check the pull request #38 Made. I have added a dice game using html,css & javascript. Please merge the PR and provide labels to it.

**Comment by Hozipitol** [comment]
@AbhilashaGupta16 I have added my folder seat_booking (request #48 ).Kindly review it andd merge if satisfied

**Comment by anushavc** [comment]
I have made a temperature convertor site using html,css and js. Could you please review it and merge it if satisfied?
